### ISSUE TITLE
Train for an unconditional model

### ISSUE THREAD

**Issue by LouisRouss** [lllyasviel/ControlNet#373]
Hi !

I'm currently trying to train a ControlNet for a different model from StableDiffusion (unconditionnal model, originally trained on ImageNet, finetuned on a specific dataset. The architecture is the one used in the guided diffusion repo ). However it appears that my ControlNet doesn't show any sign of convergence. I have already trained for 50k iterations. It seems the condition is simply ignored.

Do you have an intuition/insight on this ? Have you tried to train on a model lighter than stable diffusion ? In term of architecture of the controlnet, my model operates directly in the pixel space so the input_hint_block is a simple zero conv. 

Of course i don't rule out a mistake in my code.

Thank you for your help

**Comment by XiaoluDing** [comment]
I have the same confusion, the loss on the custom dataset has been oscillating. @LouisRouss How did you solve it?

**Comment by LouisRouss** [comment]
After multiple attempts i decided to train a model directly with the condition i needed without using a hypernetwork. The issue isn't really solved

**Comment by Benny0323** [comment]
@LouisRouss, I have the same problem.
I find the model desn't generate images according to the input condition, regardless of more than 50k iterations
Maybe the reason is that direct learning in pixel space is hard compared with latent space.
Thanks!
